examples/freetype_gl/text_rendering_2/text_rendering_2.gpr

```diff
@@ -10,7 +9,9 @@ project Text_Rendering_2 is
    for Exec_Dir use ".";
    for Main use ("render_text.adb");
 
-   package Compiler renames Opengl_Shared.Compiler;
+package Compiler is
+   for Default_Switches ("ada") use ("-gnatyctux");
+end Compiler;
```

Comment:
Why does this not rename `OpenGL.Compiler` now?

Reply:
I tried  `package Compiler renames OpenGL.Compiler;`
but the Builder fails with `"OpenGL" is not an imported or extended project`.
If you can help me fix it, should I should I change all examples to use it?

Reply:
Well you need a

    with "../../../opengl.gpr";

Unlike in Ada, the parent project is not automatically imported when you *with* `opengl-text.gpr`.

Reply:
OK. That works
I thought I tried that bit i'd copied other `limited with` lines which was my problem.
Thanks for the fix.
I'd like to use it in all examples.
Will this be OK.

Reply:
Sure.